doc/HOWTO_RELEASE.rst.txt

```diff
@@ -271,7 +271,7 @@ Mention at least the following:
   - outlook for the near future
 
 Also make sure that as soon as the branch is made, there is a new release
-notes file in trunk for the next release.
+notes file in the trunk for the next release.
```

Comment:
This should actually say `master` I think, and refers to the SVN / CVS / ??? days before git

Reply:
Since I'm still going through the docs and trying to understand the project structure, I was actually confused with the usage of this term, whether it refers to ```master``` branch only or is another development structure being followed/another technical term/ file name that I haven't went through yet. 

But, I just read more about this term used in SVN days and came across [this blog](https://www.toptal.com/software/trunk-based-development-git-flow) to understand trunk-based and git-based development workflow. Since in the current workflow in this project, we create separate branches for new features and not on a single branch as described in the above blog, I also completely agree with your views. 
So, shall I go forth with fixing the wording from ```trunk``` to ```master``` in the docs?

Reply:
yes please

Reply:
Alright, I've changed those wordings in the current file as per the context. Please kindly review them. :)